code/modules/mob/living/emote.dm

```diff
@@ -199,6 +199,22 @@
 	message = "blows a kiss."
 	message_param = "blows a kiss to %t."
 
+/datum/emote/living/kiss/run_emote(mob/living/user, params, type_override, intentional)
+	if(!user.usable_hands) // so simple mobs and such can still get in on the fun
+		return ..()
+
+	message = "" // the kisser item displays the message on firing
+	message_param = ""
```

Comment:
This is a bit of a hack, no? Shouldn't "displays a message" just be a flag?

Reply:
tbqh actually i'm just gonna remove the message entirely because if you're a handless creature who really wants to emote a kiss you can just manually type it

Reply:
and that person would be better off for it